### ISSUE TITLE
Distinguish (inflected) word-forms seen in a corpus from computational-model-generated ones

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#455]
We want to show more prominently in the paradigms (inflected) word forms that have been seen in some corpus, in comparison to (inflected) word forms that have been generated by the computational model.

This could be a way of dealing with the critique about the weirdness of non-attested forms which may be semantically or pragmatically weird but structurally sound - and possible with enough creative thinking.

We can either tag these in the computational model with a feature like `+Obs`, or create a list of all such inflected word forms. So this may require linguistic modeling work.

**Comment by aarppe** [comment]
Following are the `Present` tense forms for _wâpamêw_ (with only _ê-_ preverb forms for the Conjunct):

```
cat ~/altlab/crk/corpora/ahenakew_wolfart_MGS_tab-sep-anls_freq-sorted.txt | egrep '\<wâpamêw\>' | gawk '{ if(index($0,"Prs")!=0) if(index($0,"PV")!=0) if(index($0,"PV/e")!=0) print; else; else print; }'
  10 ê-wâpamak	PV/e+wâpamêw+V+TA+Cnj+Prs+1Sg+3SgO
   4 ê-wâpamât	PV/e+wâpamêw+V+TA+Cnj+Prs+3Sg+4Sg/PlO
   4 niwâpamâw	wâpamêw+V+TA+Ind+Prs+1Sg+3SgO
   2 ê-wâpamakik	PV/e+wâpamêw+V+TA+Cnj+Prs+1Sg+3PlO
   1 ê-wâpamâyêk	PV/e+wâpamêw+V+TA+Cnj+Prs+2Pl+3SgO
   1 ê-wâpamitân	PV/e+wâpamêw+V+TA+Cnj+Prs+1Sg+2SgO
   1 ê-wâpamit	PV/e+wâpamêw+V+TA+Cnj+Prs+3Sg+1SgO
   1 ê-wâpamiht	PV/e+wâpamêw+V+TA+Cnj+Prs+X+3SgO
   1 ê-wâpamakwâw	PV/e+wâpamêw+V+TA+Cnj+Prs+1Sg+3PlO+Morph
   1 wâpamêwak	wâpamêw+V+TA+Ind+Prs+3Pl+4Sg/PlO
   1 wâpamêw	wâpamêw+V+TA+Ind+Prs+3Sg+4Sg/PlO
   1 niwâpamâwak	wâpamêw+V+TA+Ind+Prs+1Sg+3PlO
   1 kiwâpamâwâwak	wâpamêw+V+TA+Ind+Prs+2Pl+3PlO
   1 kiwâpamâwak	wâpamêw+V+TA+Ind+Prs+2Sg+3PlO
   1 kiwâpamâw	wâpamêw+V+TA+Ind+Prs+2Sg+3SgO
```
If we were to mark the observed forms in the basic VTA paradigm, that would look like the following:

**Comment by aarppe** [comment]
First implementation proposal:
1. Read in a word-form list with disambiguated analyses and frequencies.
2. Mark unobserved word forms in grey, in contrast to full-black for observed word-forms (or something comparable).

**Comment by eddieantonio** [comment]
Resolved by #506
